util/src/main/java/org/datacommons/util/TmcfCsvParser.java

```diff
@@ -375,6 +368,14 @@ private String newUUID() {
   }
 
   private void addLog(Debug.Log.Level level, String counter, String message) {
-    logCtx.addEntry(level, counter, message, csvFileName, csvParser.getCurrentLineNumber());
+    logCtx.addEntry(level, counter, message, csvFileName, getCurrentLineNumber());
+  }
+
+  // Returns the current line in the CSV.
+  private long getCurrentLineNumber() {
```

Comment:
nit: up to you, but if all accesses are local, why not just do this.currentLineNumber in call sites?

Reply:
normally, I'd also think YAGNI, but in this case, we did just swap out the implementation detail. this felt like enough justification to have this as a point of flexibility. also, it is "what should have been", and I actually got to write this change faster because I changed this function, rather than editing the 6 locations across the file.